src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureSelector.cpp

```diff
@@ -28,12 +28,17 @@
 // ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 // --------------------------------------------------------------------------
-// $Maintainer: Douglas McCloskey, Pasquale Domenico Colaianni $
-// $Authors: Douglas McCloskey, Pasquale Domenico Colaianni $
+// $Maintainer: Douglas McCloskey, Pasquale Domenico Colaianni, Svetlana Kutuzova $
+// $Authors: Douglas McCloskey, Pasquale Domenico Colaianni, Svetlana Kutuzova $
 // --------------------------------------------------------------------------
 
+#include <algorithm>
+#include <unordered_set>
+#include <OpenMS/KERNEL/FeatureMap.h>
+#include <OpenMS/DATASTRUCTURES/LPWrapper.h>
```

Comment:
Currently _LPWrapper_ segfaults in OpenMS `develop`. It's been like that for a long time, and maybe it doesn't happen in older versions of gcc (I think I am using version 7 and you have version 8).
This is suspicious and, since we depend on that class, it probably makes sense to look into it.